src/_nebari/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/main.tf

```diff
@@ -209,3 +211,18 @@ module "jupyterhub-openid-client" {
   ]
   jupyterlab_profiles_mapper = true
 }
+
+
+resource "kubernetes_secret" "argo-workflows-conda-store-token" {
+  metadata {
+    name      = "argo-workflows-conda-store-token"
+    namespace = var.namespace
+  }
+
+  data = {
+    "conda-store-api-token"    = var.conda-store-argo-workflows-jupyter-scheduler-token
+    "conda-store-service-name" = var.conda-store-service-name
```

Comment:
This is a url I guess.  I want to double check this later and make sure there's not a better way.

Reply:
yeah this is an internal service url. `conda-store-service-name`, as a variable name, is used elsewhere in the other services so I just kept the same name.